examples/sod-mpi.py

```diff
@@ -100,11 +120,22 @@ def main(ctx_factory=cl.create_some_context):
     discr = EagerDGDiscretization(
         actx, local_mesh, order=order, mpi_communicator=comm
     )
+
     nodes = thaw(actx, discr.nodes())
     current_state = initializer(0, nodes)
 
-    visualizer = make_visualizer(discr, discr.order + 3
-                                 if discr.dim == 2 else discr.order)
+    if use_logmgr:
+        logmgr.add_quantity(PhysicalQuantity(discr, eos, "pressure", "P", "min"))
+        logmgr.add_quantity(PhysicalQuantity(discr, eos, "temperature", "K", "min"))
+        if rank == 0:
+            logmgr.add_watches(["step", "t_step", "min_pressure", "min_temperature"])
+        if use_profiling:
+            logmgr.add_quantity(KernelProfile(actx, "diff", "flops"))
```

Comment:
I think this is impossible to use. How is a user going to know what all the relevant kernel names are? These should be added automatically.

Reply:
Kernels are now added automatically.